fitwsarah_frontend/src/views/AppointmentPage/Calendar.js

```diff
@@ -57,48 +106,27 @@ const AvailabilitiesCalendar = ({onChange}) => {
-                            <label class="" for="timeslot_1" tabindex="-1"><span>10:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_2" name="selectedTimeSlot" onChange={handleTimeChange} value="11:00"/>
-                            <label class="" for="timeslot_2" tabindex="-1"><span>11:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_3" name="selectedTimeSlot" onChange={handleTimeChange} value="12:00"/>
-                            <label class="" for="timeslot_3" tabindex="-1"><span>12:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_4" name="selectedTimeSlot" onChange={handleTimeChange} value="13:00"/>
-                            <label class="" for="timeslot_4" tabindex="-1"><span>13:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_5" name="selectedTimeSlot" onChange={handleTimeChange} value="14:00"/>
-                            <label class="" for="timeslot_5" tabindex="-1"><span>14:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_6" name="selectedTimeSlot" onChange={handleTimeChange} value="15:00"/>
-                            <label class="" for="timeslot_6" tabindex="-1"><span>15:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_7" name="selectedTimeSlot" onChange={handleTimeChange} value="16:00"/>
-                            <label class="" for="timeslot_7" tabindex="-1"><span>16:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_8" name="selectedTimeSlot" onChange={handleTimeChange} value="17:00"/>
-                            <label class="" for="timeslot_8" tabindex="-1"><span>17:00</span><br/></label>
-                        </li>
-                        <li class="" >
-                            <input type="radio" tabindex="-1" id="timeslot_9" name="selectedTimeSlot" onChange={handleTimeChange} value="18:00"/>
-                            <label class="" for="timeslot_9" tabindex="-1"><span>18:00</span><br/></label>
-                        </li>
-                    </ul>
+            {!noAvailabilities &&
+                      <ul>
+                        
+                        {availabilities.slice().sort((a, b) => {
```

Comment:
Can u pls elaborate on what this line of code does and what this method is trying to do.

Reply:
Good observation! This code allows me to sort through all the availabilities meaning it will render the availabilities in an order from earlier to later. This will make it clearer for both the personal trainer and the members.